src/BenchmarksApps/TodosApi/TodosApi.http

```diff
@@ -1,6 +1,6 @@
 @TodosApi_HostAddress = http://localhost:5054
 # Uncomment following line if sending requests to published app
-#@TodosApi_HostAddress = http://localhost:5000
+@TodosApi_HostAddress = http://localhost:5000
```

Comment:
```suggestion
#@TodosApi_HostAddress = http://localhost:5000
```

If we are going to leave the `# Uncomment following line if sending requests to published app` line in, we might as well leave this line commented out.